src/test/scala/no/ndla/draftapi/integration/ConceptApiClientTest.scala

```diff
@@ -0,0 +1,47 @@
+package no.ndla.draftapi.integration
+
+import no.ndla.draftapi.{TestEnvironment, UnitSuite}
+import org.mockito.ArgumentMatchers._
+
+import scala.util.Success
+
+class ConceptApiClientTest extends UnitSuite with TestEnvironment {
+  val baseUrl = "http://mockpath"
+  val draftPath = s"concept-api/v1/drafts"
+  val idPath = (id: Long) => s"$draftPath/$id"
+
+  override val conceptApiClient: ConceptApiClient = spy(new ConceptApiClient(baseUrl))
+
+  override protected def beforeEach(): Unit = {
+    // Since we use spy, we reset the mock before each test allowing verify to be accurate
+    reset(conceptApiClient)
+  }
+
+  test("Test that publishConceptsIfToPublishing publishes concepts with QUEUED_FOR_PUBLISHING status") {
+    val idPath1 = s"$draftPath/100"
+    val idPath2 = s"$draftPath/200"
+
+    val concept = DraftConcept(
+      id = 1,
+      status = ConceptStatus(
+        current = "QUEUED_FOR_PUBLISHING"
```

Comment:
Skulle det egentlig vore PUBLISHED her for at returverdiene i kvar doReturn skal gi meining?

Reply:
Den biten her var egentlig bare for å ha noe å kopiere.
Endte jo med å ikke bruke noen av feltene fra orignalen, så kan vel egentlig lage nye direkte.

Reply:
Ja for du bruker jo ikkje returverdiene. Sånn det står no så returnerer publiseringskallet et concept med status QUEUED_FOR_PUBLISHING men fordi du ikkje sjekker at denne statusen er PUBLISHED så har det ingenting å sei.

Reply:
åja du tenker på etter kallet til `PUT` endepunktet?
Ja, har nok ikke så mye å si, men kan endre det siden det er det som gir mest mening.